install/disconnected_install.adoc

```diff
@@ -247,11 +247,11 @@ endif::[]
 [source, bash]
 ----
 $ docker pull registry.access.redhat.com/openshift3/logging-auth-proxy:<tag>
```

Comment:
We no longer use the `logging-auth-proxy`

Reply:
@ewolinetz Is the change to drop `logging-auth-proxy` in 3.11+?